capstone/src/main/webapp/multilinevisualization.js

```diff
@@ -0,0 +1,225 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const d3 = window.d3;
+/**
+ * Setup dummy data for sample visualization
+ */
+const dateArr = [
+  new Date(2020, 2, 1),
+  new Date(2020, 2, 8),
+  new Date(2020, 2, 15),
+  new Date(2020, 2, 22),
+  new Date(2020, 2, 29),
+  new Date(2020, 3, 5),
+  new Date(2020, 3, 12),
+];
+const data = {
```

Comment:
Since you will be getting the real data from a servlet, I recommend that rather than doing "const", you stick this dummy data into a "getFooBarWhatever" function that returns it, so you force yourself to write you SVG use in a modular way that doesn't rely on data being there already when the page loads.

Reply:
I believe that makes sense; I've created a function for it. Let me know if I got the right understanding!